src/backends/caffe/caffelib.cc

```diff
@@ -208,25 +208,22 @@ namespace dd
 	caffe::ReadProtoFromTextFile(dest_net,&net_param); //TODO: catch parsing error (returns bool true on success)
 	caffe::ReadProtoFromTextFile(dest_deploy_net,&deploy_net_param);
 
-       if (this->_loss.compare(0, 4, "dice",0,4) == 0)
+       if (this->_loss == "dice")
       // dice loss!!
       {
-        int ignore_label = -1;
-        if (ad.has("ignore_label"))
```

Comment:
I don't believe we should remove `ignore_label`. In practice it is very useful with at least the softmax and weighted softmax losses, to ignore some classes. One drawback is that we can ignore one class only in the current implementation, which is due to Caffe initial implementation.

Reply:
In case of dice loss, I have not been able so far to find a satisfactory way to ignore a class despite having tested several ways to do so. I will put that in my todo list

Reply:
Ok but it should remain available for the softmax loss, right ?

Reply:
I think it does , we are inside the if (this->_loss == "dice")   block   if I do not fool myself